fact_extractor/install/pre_install.sh

```diff
@@ -4,7 +4,7 @@ echo "Install Pre-Install Requirements"
 (apt-get update && apt-get install sudo) || true
 
 sudo apt-get update
-sudo apt-get -y install git apt-transport-https ca-certificates curl software-properties-common wget libmagic-dev
+sudo apt-get -y install git apt-transport-https ca-certificates curl software-properties-common wget libmagic-dev rustc cargo
```

Comment:
this should (hopefully) not be necessary any more with the pre-built wheels of entropython for aarch64